lib/umwelt/client/fetcher.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Umwelt.Client.Fetcher do
+  @moduledoc "Code fetcher task"
+
+  use Task
+  require Logger
+
+  alias Umwelt.Client
+
+  def start_link(module) do
+    Task.start_link(__MODULE__, :run, [module])
```

Comment:
Technically, they recommend [`Task.Supervisor.start_child/3`](https://hexdocs.pm/elixir/Task.Supervisor.html#start_child/3) to delegate supervising to the already existing code in _Elixir_ core.